Cargo.toml

```diff
@@ -268,9 +270,17 @@ pallet-evm-precompile-bridge-transfer = { path = "precompiles/bridge-transfer",
 pallet-evm-precompile-parachain-staking = { path = "precompiles/parachain-staking", default-features = false }
 pallet-evm-precompile-score-staking = { path = "precompiles/score-staking", default-features = false }
 pallet-evm-assertions = { path = "pallets/evm-assertions", default-features = false }
-# This is a temporary pallet copy from Astar with polkadot-0.9.43
-# The official release of this pallet will be in frontier repo
-precompile-utils = { git = "https://github.com/litentry/precompile-utils", branch = "main", default-features = false }
 
 [patch.crates-io]
 ring = { git = "https://github.com/betrusted-io/ring-xous", branch = "0.16.20-cleanup" }
+
+[patch."https://github.com/AstarNetwork/frontier"]
```

Comment:
I don't find anywhere else using `fc-consensus` from `https://github.com/AstarNetwork/frontier`. Why do we need this patch?

Reply:
Can be indirect dep